databricks/koalas/series.py

```diff
@@ -4730,6 +4730,30 @@ def asof(self, where):
         result_series.name = self.name
         return result_series
 
+    def item(self):
+        """
+        Return the first element of the underlying data as a python scalar.
```

Comment:
python -> Python

Reply:
@HyukjinKwon Ah, I just followed the docstring of pandas, but It seems better to fix. Thanks!